website/docs/reference/project-collaboration-mode.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Project collaboration mode
+---
+
+:::info Availability
+
+The change requests feature is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+
+:::
+
+Unleash project collaboration mode is a feature that allows to specify who can submit [change requests](change-requests.md). Currently, there are two modes:
+- **open** where everyone can submit change requests. This is a default setting for the existing and new Unleash projects.
+- **protected** where [admins, project members and owners](rbac.md#standard-roles) can submit change requests. Users who are not added to projects will not be able to submit change requests.
+
+## Project collaboration mode setting
+
+Project collaboration mode is controlled from the "Create Project" and "Edit Project" screens.
+
+![Create project with collaboration mode](/img/collaboration-mode.png)
+
+When project collaboration mode is protected all users who are not project members or admins will have all the
+change request operations disabled.
+
+## Enabling protected mode for an existing project
+
+To change the project collaboration mode for an existing project you have to edit the project.
+
+![Edit project](/img/edit-project.png)
```

Comment:
Right, because this is one of the few settings that are available from "edit project" instead of "project settings"? What's the reason for that? And why do we have those two separate sections anyway? 

I've added a suggestion for this link higher up, so I'll add one to remove this line too.

```suggestion
```

Reply:
We wanted to delivery this capability very soon and it was the quickest and deliberate route. Eventually we'll create project settings in the next quarter.